standalone-metastore/src/main/thrift/hive_metastore.thrift

```diff
@@ -729,6 +732,15 @@ struct AbortTxnsRequest {
 
 struct CommitTxnRequest {
     1: required i64 txnid,
+    2: optional string replPolicy,
+}
+
+struct GetTargetTxnIdRequest {
```

Comment:
Unused GetTargetTxnIdRequest/GetTargetTxnIdResponse.

Reply:
this will be used for alloc write id event ..anyways removed it from this patch